### DISCUSSION TITLE
Magnetic Bearings parameter issue

### DISCUSSION THREAD

**Discussion by engfrancisco** [petrobras/ross#812] [Q&A] [answered]
Hello, I was studying and verifying the implementation of magnetic bearings and I believe there was a mistake in a parameter of the damping equation, as below

![Magnetic_Bearing](https://user-images.githubusercontent.com/67988135/123848652-5718ef00-d8ee-11eb-9f8e-ffa26ddaf312.PNG)

The parameter pA[5] is the kp_pid, however, in the damping equation, we have the kd_pid, that is, the pA[6] parameter. I verified the book reference and the equation it's the same, as the commented equation

The function are "bearing_seal_element.py" at "ross" paste

**Comment by davidjgm** [comment, marked as answer]
Hi @engfrancisco.
Thanks for letting us know the mistake. I already made the Pull Request to correct this issue.

**Reply by engfrancisco** [reply]
Thank you!
